kmp-library/src/commonMain/kotlin/com/novoda/aws/v4/signer/hash/Sha256.kt

```diff
@@ -1,5 +1,7 @@
 package com.novoda.aws.v4.signer.hash
 
+import kotlinx.serialization.toUtf8Bytes
```

Comment:
This is part of `io.ktor:ktor-client-json`, so we get rid of our custom extension method.